### PR TITLE
Revise/edit overall design in docs overview*.rst

### PR DESCRIPTION
**Pull request by bernstei**
Revised to clean up language and add a few details.  It also refers to the syntax in the new more decorator-like `autoparallelize` (previously `iloop`), so it probably shouldn't be merged until #108 is.

### CODE REVIEW

File: docs/source/overview.rst [hunk 1]
```diff
@@ -1,19 +1,57 @@
...
+and write ``Atoms`` to ``OutputSpec``, regardless of how the input
+configs were supplied or how or where to the output configs are going
+to be collected.
+
+In addition to this abstraction of ``Atoms`` storage, the workflow makes
+it easy to parallelize operations over sets of configurations and/or
+run them as (possibly remote) queued jobs, and this has been implemented
+for most of its operations.  This is achieved by wrapping the operation in a
+call to ``wfl.pipeline.autoparallelize``. In addition to parallelising
+on readily accessible cores, the operations may executed in a number
```
Comment (gelzinyte) [inline docs/source/overview.rst:45]: "may be executed"?

File: docs/source/overview.rst [hunk 2]
```diff
@@ -1,19 +1,57 @@
 .. _code_structure:
 
 ######################################
-Code structure
+Overall design
 ######################################
 
-Most of :ref:`operations <operations>` in Workflow take in (optional) and return (mandatory) an ASE's ``Atoms`` object. These inputs and outputs are abstracted in ``ConfigSet`` and ``OutputSpec`` classes.  ``ConfigSet`` may be initialised with a list(s) of ``Atoms``, by providing one or multiple ``.xyz`` filenames or with a reference to an `ABCD database <https://github.com/libAtoms/abcd>`_. Similarly, returned configurations can be held in memory and subsequently used via ``OutputSpec.to_ConfigSet()``, written to a single or multiple files or uploaded to an ABCD database. This way, an operation may iterate over a ``ConfigSet`` and write ``Atoms`` to ``OutputSpec``, regardless of how the input configs were supplied or how or where to the output configs are going to be collected.
+The main goal of the workflow is make it easy operate on sets of input atomic
```
Comment (gelzinyte) [inline docs/source/overview.rst:7]: "main goal of Workflow" or "main goal of the Workflow package"? Also - "is **to** make it easy **to** operate"?

**Review by gelzinyte — APPROVED** [review]
Looks great, thanks! Do merge when you wish.

File: docs/source/overview.rst (hunk 1 above)
Comment (bernstei) [reply docs/source/overview.rst:45]: yup

**Pull request merged**